packages/Activity-Tracker-browser-extension/storage/server.js

```diff
@@ -0,0 +1,24 @@
+import { post } from 'axios';
+// const express = require('express');
+
+const saveActivities = activity => {
+	const { title, url, favicon, startTime, endTime } = activity;
+
+	post('http://localhost:32768/api/browseractivities', {
```

Comment:
Use fetch instead of axios. Fetch is directly available in the browser so we dont have to bundle axios with it.

Reply:
okey

Reply:
@supercoww I don't think this storage file is need in `packages/Activity-Tracker-browser-extension/storage/server.js` as I am already sending it in background.js

Reply:
Ok. remove it.